docs/concepts.rst

```diff
@@ -1,128 +1,130 @@
+ 
+**Managed packages** are typically used by Salesforce partners to distribute and sell applications to customers. They are proprietary code that can only be upgraded and deployed by the developer that built them. To ensure seamless upgrades, certain destructive changes, like removing objects or fields, cannot be performed.
 
-Throughout the CumulusCI documentation, we'll have the Product Delivery Model in mind. 
+In CumulusCI, packages are also built, stored and deployed via **projects**.
 
 Projects
 --------
 
-When you work with CumulusCI, you do so inside a project. A project is a version control repository that contains Salesforce metadata as well as the CumulusCI automation that builds and releases the project. A project usually has a one-to-one relationship with a managed package, if building a package, or with an org implementation. A project may constitute the entirety of a product in the Product Delivery Model, or may be one of multiple projects making up a product.
+When you work with CumulusCI, you do so inside a project. A project is an individual git repository (think GitHub) that contains Salesforce metadata as well as the CumulusCI automation (e.g., tasks and flows) that builds and releases the project. Most importantly, a project can build one - and only one(!) - package in its repository. This one-to-one relationship between project and package is what allows version control to track any changes your team made to the repository with minimal collision.
 
-CumulusCI scopes many of its activities to the project, so you'll always run your CumulusCI commands inside your repository directory. Each project gets its own keychain, which we'll talk about in detail below. Each project has its own set of scratch orgs, which CumulusCI namespaces in the Salesforce CLI keychain to prevent collisions. And each project is linked to a single GitHub repository, so you can always be confident of the scope of the actions you're taking.
-
-CumulusCI knows which project you are working on by the current working directory of your shell. Make sure to change directories inside your project before you run project-specific commands.
-
-In order to be used as a CumulusCI project, a directory must both be a Git repository and contain a ``cumulusci.yml`` configuration file. We'll show you how to get set up with a new or existing CumulusCI project in the (TODO link) Get Started section.
+It's important to note that a project may not even contain a package; for example, it can be built to deliver functionality to records, or test data for QA. A project may constitute the entirety of a product made for release, or may be one of multiple projects banded together to make a product. Basically, whatever feature in your product that needs specialized Salesforce metadata and CumulusCI automation in order to be built and deployed, *especially* packages (we cannot stress this enough!), requires its own unique project. 
```

Comment:
@lianeyee submitted her edit on this, too, so I'm tagging her here so she can have eyes on it. 

@davidmreed I really like your version because it's even more succinct. (I'll swap out the "may".) 

That said, "builds either one or zero packages" is true, but it's not a great selling point for cci. My reason for restructuring the entirety of Key Concepts is to center everything around packages, which is the primary motivator for someone to use CCI in the first place, correct? So I think it's prudent to emphasize that while a project's existence doesn't hinge on whether or not it contains a package, it _is_ the vehicle for a package.

I suggest we end this section with this:

"To sum up, although a project doesn't require a package, a package requires a project to be built and deployed."